<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EMMCWidget</class>
 <widget class="QWidget" name="EMMCWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_boot1Size">
           <property name="text">
            <string>Boot1 Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_boot2Size">
           <property name="text">
            <string>Boot2 Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_rpmbSize">
           <property name="text">
            <string>RPMB Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_gp1Size">
           <property name="text">
            <string>GP1 Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_gp2Size">
           <property name="text">
            <string>GP2 Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_gp3Size">
           <property name="text">
            <string>GP3 Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_gp4Size">
           <property name="text">
            <string>GP4 Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_uaSize">
           <property name="text">
            <string>UA Size:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_boot1">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_boot2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_rpmb">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_gp1">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_gp2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_gp3">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_gp4">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ua">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
